akka-docs/src/test/java/jdocs/stream/operators/SinkDocExamples.java

```diff
@@ -31,5 +34,50 @@ static void reduceExample() throws InterruptedException, ExecutionException, Tim
         // 55
         //#reduce-operator-example
     }
-}
 
+    static void takeLastExample() throws InterruptedException, ExecutionException, TimeoutException {
+        //#takeLast-operator-example
+        // pair of (Name, GPA)
+        List<Pair> sortedStudents = Arrays.asList(new Pair<>("Benita", 2.1), new Pair<>("Adrian", 3.1),
+                new Pair<>("Alexis", 4), new Pair<>("Kendra", 4.2), new Pair<>("Jerrie", 4.3), new Pair<>("Alison", 4.7));
+
+        Source<Pair, NotUsed> studentSource = Source.from(sortedStudents);
+
+        CompletionStage<List<Pair>> topThree = studentSource.runWith(Sink.takeLast(3), materializer);
+
+        List<Pair> result = topThree.toCompletableFuture().get(3, TimeUnit.SECONDS);
```

Comment:
Instead of using `get`, perhaps we could show `thenAccept(result -> { ... })` to avoid showing blocking in the docs?

Reply:
done.